package.json

```diff
@@ -13,10 +13,13 @@
     "winston": "^3.2.1"
   },
   "devDependencies": {
+    "jest": "^26.0.1",
     "nodemon": "^2.0.2"
   },
   "scripts": {
     "start": "node index.js",
-    "dev": "nodemon index.js"
+    "dev": "nodemon index.js",
+    "jest": "jest clearCache && jest",
```

Comment:
Can we rename this script form jest to test? That's a more common practice for Node projects.

Reply:
@iamvery Asked the very same thing on #39.

I'll update it in both branches/PRs.

Reply:
I've changed the name on #39 and have merged that branch onto this one.